test-cases/gemini/gemini-3h-ics-cdc-with-nemesis.yaml

```diff
@@ -6,6 +6,7 @@ n_monitor_nodes: 1
 instance_type_db: 'i3.large'
 
 user_prefix: 'ics-cdc-gemini-basic-3h'
+ami_db_scylla_user: 'centos'
```

Comment:
if the system under test is running on Ubuntu, it will break the test, no?

Reply:
Not with our own AMIs/Images, we have an alias named centos on all of them.

Reply:
but in any case, it seems to be wrong, to change it globally because of the oracle node having a different username setting

Reply:
> but in any case, it seems to be wrong, to change it globally because of the oracle node having a different username setting

Where do you see "global" change? One particular config is changed here and it is used in only 1 (one) jenkins job.

Reply:
the meaning is that it is global for that specific test, and it may be wrong depending on the AMI you will use...
if your AMI is based on anything other than RHEL, it may not work (for this test), as the default user for Ubuntu based AMIs is `ubuntu`.. i know we have an alias that might be common to all AMIs, and we should use it?

Reply:
we have `scyllaadm` which should be the common, problem is that 2020.1 AMIs were before that.
I think we probably better off with setting the oracle to 2021.1

Reply:
yes, as 2021.1 is supposed to be stable (current version is 2021.1.10)